### PR TITLE
Update BCD info, part 6

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #19185 

The PR updates BCD info in WebAPI docs.

It focuses on files that have only left behind 'experimental' tags and headers.

### CODE REVIEW

File: files/en-us/web/api/audioparammap/index.md [hunk 1]
```diff
@@ -4,7 +4,6 @@ slug: Web/API/AudioParamMap
 page-type: web-api-interface
 tags:
   - Draft
```
Comment (teoli2003) [inline files/en-us/web/api/audioparammap/index.md:6]:
```suggestion
  - Interface
```

File: files/en-us/web/api/bluetoothremotegattcharacteristic/writevalue/index.md [hunk 2]
```diff
@@ -6,7 +6,6 @@ tags:
   - API
   - Bluetooth
   - BluetoothRemoteGATTCharacteristic
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothremotegattcharacteristic/writevalue/index.md:9]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md [hunk 3]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Method
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md:9]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md [hunk 4]
```diff
@@ -6,12 +6,11 @@ tags:
...
   - canonicalUUID()
   - BluetoothUUID
 browser-compat: api.BluetoothUUID.canonicalUUID
 ---
-{{APIRef("Bluetooth API")}}{{SeeCompatTable}}
+{{APIRef("Bluetooth API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md:13]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/getcharacteristic/index.md [hunk 5]
```diff
@@ -6,12 +6,11 @@ tags:
   - API
   - Method
   - Reference
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/getcharacteristic/index.md:9]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/getcharacteristic/index.md [hunk 6]
```diff
@@ -6,12 +6,11 @@ tags:
...
   - getCharacteristic
   - BluetoothUUID
 browser-compat: api.BluetoothUUID.getCharacteristic
 ---
-{{APIRef("Bluetooth API")}}{{SeeCompatTable}}
+{{APIRef("Bluetooth API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/getcharacteristic/index.md:13]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/getdescriptor/index.md [hunk 7]
```diff
@@ -7,11 +7,10 @@ tags:
   - Method
   - Reference
   - getDescriptor
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/getdescriptor/index.md:10]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/getdescriptor/index.md [hunk 8]
```diff
@@ -7,11 +7,10 @@ tags:
...
   - BluetoothUUID
 browser-compat: api.BluetoothUUID.getDescriptor
 ---
-{{APIRef("Bluetooth API")}}{{SeeCompatTable}}
+{{APIRef("Bluetooth API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/getdescriptor/index.md:13]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/getservice/index.md [hunk 9]
```diff
@@ -7,11 +7,10 @@ tags:
   - Method
   - Reference
   - getService
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/getservice/index.md:10]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

File: files/en-us/web/api/bluetoothuuid/getservice/index.md [hunk 10]
```diff
@@ -7,11 +7,10 @@ tags:
...
   - BluetoothUUID
 browser-compat: api.BluetoothUUID.getService
 ---
-{{APIRef("Bluetooth API")}}{{SeeCompatTable}}
+{{APIRef("Bluetooth API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/getservice/index.md:13]: This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
> This is the opposite: bcd is wrong. This is supported by one engine. So we should have the experimental tag and the macro here, and add the flag on bcd.

Good catch!
I've updated the files. And here is the `mdn/browser-compat-data` [PR](https://github.com/mdn/browser-compat-data/pull/17288)

**Comment by OnkarRuikar** [comment]
Regarding: `files/en-us/web/api/bluetoothremotegattcharacteristic/writevalue/index.md`
Looks like it can't have deprecated and experimental statuses at the same time. The browser-compat-data build fails with following error:
```
Testing all features together...
Status - 1 problem (1 error, 0 warnings):
 ✖ api.BluetoothRemoteGATTCharacteristic.writeValue - Error → Unexpected simultaneous experimental and deprecated status in 
   ◆ Tip: Run npm run fix to fix this problem automatically
Error: Process completed with exit code 1.
```

May I remove the experimental from it?

**Comment by teoli2003** [comment]
> Regarding: `files/en-us/web/api/bluetoothremotegattcharacteristic/writevalue/index.md` Looks like it can't have deprecated and experimental statuses at the same time. The browser-compat-data build fails with following error:
> 
> ```
> Testing all features together...
> Status - 1 problem (1 error, 0 warnings):
>  ✖ api.BluetoothRemoteGATTCharacteristic.writeValue - Error → Unexpected simultaneous experimental and deprecated status in 
>    ◆ Tip: Run npm run fix to fix this problem automatically
> Error: Process completed with exit code 1.
> ```
> 
> For more info refer https://github.com/mdn/browser-compat-data/blob/989c1242912131d1a7788f360c5f2462c2d2cfe5/test/linter/test-status.ts#L81

Put only deprecated on it.

**Comment by OnkarRuikar** [comment]
> Put only deprecated on it.

Done in commit [`remove experimental from deprecated api`](https://github.com/mdn/content/pull/19340/commits/e2dbeca1bb03df1dc240967a307f03ea3761cdf9).

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**